documentation/modules/configuring/con-config-high-volume-messages.adoc

```diff
@@ -0,0 +1,216 @@
+
+* xref:con-producer-config-properties-throughput-{context}[] 
+* xref:con-consumer-config-properties-throughput-{context}[]  
+
+You can also use the same configuration options with the producers and consumers used by Kafka Connect runtime and MirrorMaker 2.0 source connectors and sink connectors.
+
+Source connectors:: 
+* Consumers retrieve messages from a source Kafka cluster for MirrorMaker 2.0. As Kafka Connect fetches data from an external data system, consumers are not usually required for source connectors.  
+* Producers send messages to a target Kafka cluster for Kafka Connect and MirrorMaker 2.0.
+
+Sink connectors:: 
+* Consumers retrieve data from a source Kafka cluster for Kafka Connect.
+* Producers push data to an external data system for Kafka Connect.
+
+For consumers, you might increase the amount of data fetched in a single fetch request to reduce latency.
+You increase the fetch request size using the `fetch.max.bytes` and `max.partition.fetch.bytes` properties.
+You can also set a maximum limit on the number of messages returned from the consumer buffer using the `max.poll.records` property.
+
+For MirrorMaker 2.0, configure the `fetch.max.bytes`, `max.partition.fetch.bytes`, and `max.poll.records` values at the source connector level (`consumer.*`), as they relate to the specific consumer that fetches messages from the source.
+
+For producers, you might increase the size of the message batches sent in a single produce request.
+You increase the batch size using the `batch.size` property.
+A larger batch size reduces the number of outstanding messages ready to be sent and the size of the backlog in the message queue.
+Messages being sent to the same partition are batched together.
+A produce request is sent to the target cluster when the batch size is reached.
+By increasing the batch size, produce requests are delayed and more messages are added to the batch and sent to brokers at the same time.  
+This can improve throughput when you have just a few topic partitions that handle large numbers of messages.  
+
+Consider the number and size of the records that the producer handles for a suitable producer batch size. 
+
+Use `linger.ms` to add a wait time in milliseconds to delay produce requests when producer load decreases. 
+The delay means that more records can be added to batches if they are under the maximum batch size.  
+
+Configure the `batch.size` and `linger.ms` values at the source connector level (`producer.override.*`), as they relate to the specific producer that sends messages to the target Kafka cluster.
+
+For Kafka Connect, the data streaming pipeline to the target Kafka cluster is as follows:
+
+.Data streaming pipeline for Kafka Connect
+*external data source -> (Kafka Connect tasks) source message queue -> producer buffer -> target topic* 
+
+For MirrorMaker 2.0, the data mirroring pipeline to the target Kafka cluster is as follows:
+
+.Data mirroring pipeline for MirrorMaker 2.0
```

Comment:
The titles are fine now from my point of view. But it seems a bit weird that you select out source connectors and MM2 but you do not have the same for sink connectors (and their consumer clients).

Reply:
I've added a sink connector flow. 
This ties in nicely with the new sink connector example.